modalapi/mod.py

```diff
@@ -774,7 +776,14 @@ def system_disable_hotspot(self):
 
     def system_enable_hotspot(self):
         self.system_toggle_hotspot("Enabling, please wait...", "/usr/bin/patchbox wifi hotspot up")
-
+        
+    def system_updater(self):
+        self.update_system("Updating system, please wait...", "/home/patch/pi-stomp/modalapi/update.sh")
```

Comment:
We should get rid of all those hard wired path... maybe dig them out of config, so we can one day start installing properly